lldb/include/lldb/Core/ModuleList.h

```diff
@@ -285,7 +285,7 @@ class ModuleList {
   /// \see Module::FindFunctions ()
   void FindFunctions(ConstString name, lldb::FunctionNameType name_type_mask,
                      const ModuleFunctionSearchOptions &options,
-                     SymbolContextList &sc_list) const;
+                     const SymbolContext &sc, SymbolContextList &sc_list) const;
```

Comment:
How about we call `sc`, `search_first` instead, or something more self-documenting like that? (which is what `FindTypes` also does).

And as Greg pointed out, could we add documentation to the function describing what the parameter does?

Reply:
> How about we call sc, search_first instead, or something more self-documenting like that? (which is what FindTypes also does).
The word "hint" came up a couple times in the discussion as well. Maybe `search_hint` ?

> could we add documentation to the function describing what the parameter does?
Sure.